docs/integration/driver/opengles.rst

```diff
@@ -63,3 +63,43 @@ Usage
 
         return 0;
     }
+
+Custom Textures
+---------------
+
+The OpenGL driver can draw textures from the user. A third-party library could be
+used to add content to a texture and the driver will draw the texture in the window
+with the LVGL objects.
+
+for this example, copy tests/src/test_assets/test_arc_bg.c to your project directory
+and ensure ``LV_COLOR_DEPTH`` is set to ``32``.
```

Comment:
Great catch. There's another issue. `#include "../../../lvgl.h"` in test_arc_bg.c is problematic. I just switched to using an image asset from the examples instead of the tests.